### PR TITLE
Remove whitespace and use `dirname` in `mkdir` prefix in collect_image_names

### PR DESCRIPTION
**Pull request by fpopic**

### CODE REVIEW

**Review by bpietraga — DISMISSED** [review]
Looks like tidy fix 👍

File: src/docker/commands/collect_image_names.yaml [hunk 1]
```diff
@@ -6,9 +6,9 @@ parameters:
     type: string
     default: $CHANGE_NAME
 steps:
-  - run: mkdir -p tmp/image-names
-  - run: echo "<<parameters.image>>:<<parameters.tag>> " >> tmp/image-names/${CIRCLE_JOB}.txt
+  - run: mkdir -p /tmp/image-names
+  - run: echo "<<parameters.image>>:<<parameters.tag>>" >> /tmp/image-names/"${CIRCLE_JOB}".txt
```
Comment (bpietraga) [inline src/docker/commands/collect_image_names.yaml:10]: quotes here are redundant

File: src/docker/commands/collect_image_names.yaml [hunk 2]
```diff
@@ -6,8 +6,8 @@ parameters:
     type: string
     default: $CHANGE_NAME
 steps:
-  - run: mkdir -p tmp/image-names
-  - run: echo "<<parameters.image>>:<<parameters.tag>> " >> tmp/image-names/${CIRCLE_JOB}.txt
+  - run: mkdir -p "$(dirname tmp/image-names/"${CIRCLE_JOB}".txt)"
+  - run: echo "<<parameters.image>>:<<parameters.tag>>" >> tmp/image-names/"${CIRCLE_JOB}".txt
```
Comment (fpopic) [inline src/docker/commands/collect_image_names.yaml:10]:
```suggestion
  - run: echo "<<parameters.image>>:<<parameters.tag>>" >> tmp/image-names/${CIRCLE_JOB}.txt
```

**Review by bpietraga — APPROVED** [review]
🎊 🎉

**Pull request merged**